hermes-console/test/unit/AvroNodeControllerTest.js

```diff
@@ -0,0 +1,429 @@
+describe("AvroNodeController", function () {
+
+    var $controller;
+    var $scope;
+
+    beforeEach(module('hermes.topic.avroViewer'));
+
+    beforeEach(inject(function ($rootScope, _$controller_) {
+        $controller = _$controller_;
+        $scope = $rootScope;
+    }));
+
+    [
```

Comment:
#minor
Does it support [aliases](https://avro.apache.org/docs/1.8.2/spec.html#Aliases)?

Reply:
Nope, only the `name` field is displayed.